File: src/main/java/um/haberes/report/model/haberes/core/CodigoDto.java
package um.haberes.report.model.haberes.core;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import um.haberes.report.util.Jsonifyable;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CodigoDto implements Jsonifyable {

    private Integer codigoId;

    private String nombre = "";

    private byte docente = 0;

    private byte noDocente = 0;

    private byte transferible = 0;

    private byte incluidoEtec = 0;

    private Long afipConceptoSueldoIdPrimerSemestre;

    private Long afipConceptoSueldoIdSegundoSemestre;

    private AfipConceptoSueldoDto afipConceptoSueldoPrimerSemestre;

    private AfipConceptoSueldoDto afipConceptoSueldoSegundoSemestre;

}

File: src/main/java/um/haberes/report/kotlin/dto/haberes/core/CodigoDto.kt
package um.haberes.report.kotlin.dto.haberes.core

data class CodigoDto(

    var codigoId: Int? = null,
    var nombre: String = "",
    var docente: Byte = 0,
    var noDocente: Byte = 0,
    var transferible: Byte = 0,
    var incluidoEtec: Byte = 0,
    var afipConceptoSueldoIdPrimerSemestre: Long? = null,
    var afipConceptoSueldoIdSegundoSemestre: Long? = null,
    var afipConceptoSueldoPrimerSemestre: AfipConceptoSueldoDto? = null,
    var afipConceptoSueldoSegundoSemestre: AfipConceptoSueldoDto? = null,
    ) {
}
